app/src/main/java/com/infomaniak/drive/ui/home/HomeFragment.kt

```diff
@@ -67,9 +69,13 @@ class HomeFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
         mainViewModel.isInternetAvailable.observe(viewLifecycleOwner) { isInternetAvailable ->
             noNetworkCard.isGone = isInternetAvailable
         }
-
+        currentUser?.let { currentUser ->
+            if (DriveInfosController.getDrives(currentUser.id).size == 1) {
+                switchDriveButton.isEnabled = false
+                switchDriveButton.icon = null
```

Comment:
Any reason to do that instead of just hiding it, like the other one in `MenuFragment.kt`?

Reply:
Yes because here the button is the name of the current drive, with an icon attached at the end to let the user know he can switch drives. 
If we hide it, the whole name of the current drive disappear